### ISSUE TITLE
Production build filenameHashing generate the same hash every time

### ISSUE THREAD

**Issue by ealcantara22** [vuejs/vue-cli#3511]
### Version
3.4.1

### Environment info
```
    Yarn: Not Found
    npm: 6.4.1 - /usr/bin/npm
  Browsers:
    Chrome: Not Found
    Firefox: Not Found
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.0.0-beta.2 
    @vue/babel-plugin-transform-vue-jsx:  1.0.0-beta.2 
    @vue/babel-preset-app:  3.3.0 
    @vue/babel-preset-jsx:  1.0.0-beta.2 
    @vue/babel-sugar-functional-vue:  1.0.0-beta.2 
    @vue/babel-sugar-inject-h:  1.0.0-beta.2 
    @vue/babel-sugar-v-model:  1.0.0-beta.2 
    @vue/babel-sugar-v-on:  1.0.0-beta.2 
    @vue/cli-overlay:  3.3.0 
    @vue/cli-plugin-babel: ^3.0.5 => 3.3.0 
    @vue/cli-plugin-eslint: ^3.0.5 => 3.3.0 
    @vue/cli-service: ^3.0.5 => 3.3.1 
    @vue/cli-shared-utils:  3.3.0 
    @vue/component-compiler-utils:  2.5.1 
    @vue/preload-webpack-plugin:  1.1.0 
    @vue/web-component-wrapper:  1.2.0 
    babel-helper-vue-jsx-merge-props:  2.0.3 
    babel-plugin-transform-vue-jsx:  4.0.1 
    bootstrap-vue: ^2.0.0-rc.11 => 2.0.0-rc.11 
    eslint-plugin-vue: ^5.0.0-0 => 5.1.0 
    vue: ^2.5.17 => 2.5.22 
    vue-eslint-parser:  2.0.3 
    vue-functional-data-merge:  2.0.7 
    vue-hot-reload-api:  2.3.1 
    vue-loader:  15.6.1 
    vue-loading-overlay: ^3.1.0 => 3.1.1 
    vue-notification: ^1.3.13 => 1.3.14 
    vue-router: ^3.0.1 => 3.0.2 
    vue-style-loader:  4.1.2 
    vue-template-compiler: ^2.5.17 => 2.5.22 
    vue-template-es2015-compiler:  1.8.2 
    vuejs-datepicker: ^1.5.4 => 1.5.4 
    vuex: ^3.0.1 => 3.1.0 
  npmGlobalPackages:
    @vue/cli: 3.4.1
```

### Steps to reproduce
N/A

### What is expected?
Generated files for production should have different hash in order to avoid cache issues with the browser.

### What is actually happening?
The files are generated with the same name and hash always.

app.e02a4302.js

**Comment by sodatea** [comment]
Please provide a reproduction repo so that we can investigate this issue.

**Comment by sodatea** [comment]
Also, please try to delete the project lockfile and reinstall deps. Seems your local deps are of version 3.3, this issue might have already been fixed in the latest v3.4.1.

**Comment by ealcantara22** [comment]
> 
> 
> Also, please try to delete the project lockfile and reinstall deps. Seems your local deps are of version 3.3, this issue might have already been fixed in the latest v3.4.1.

That did the trick.
Thank you!!
